migrations/versions/605e99996497_.py

```diff
@@ -1,102 +0,0 @@
-"""empty message
```

Comment:
Hey @ttcollins, great job man. But you shouldn't delete migration files. that's dangerous. you are supposed to run new migrations but not delete

Reply:
Ohhh, sorry about that. Won't happen next time